docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -207,6 +207,22 @@ This means that the number of rows will drive the height of the grid and consequ
 
 {{"demo": "pages/components/data-grid/rendering/AutoHeightGrid.js", "bg": "inline"}}
 
+## Density
+
+You can change the density of the rows and the column header.
+
+### Size prop
+
+There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the `DataGrid` is set to `medium`. The default grid density can be changed by applying the `size` prop. The `size` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
```

Comment:
use the generic to not exclude XGrid.

```suggestion
There are three possible density options to choose from: `small`, `medium`, and `large`. By default, the density of the data grid is set to `medium`. The default grid density can be changed by applying the `size` prop. The `size` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
```

Reply:
```suggestion
The vertical density data grid  can be set using the `size` prop. Regardless of its setting, the `size` prop will respect the values provided using the `rowHeight` or `headerHeight` props.
```

Avoid mixing size and density terminology, avoid duplicating info from the API docs (defaults, prop values). (I don't hold a strong opinion on this second point.